lib/tilebag.rb

```diff
@@ -0,0 +1,24 @@
+module Scrabble
+  class TileBag
+
+    attr_accessor :tile_bag
+
+    def initialize
+      @tile_bag = (?A*9 + ?B*2 + ?C*2 + ?D*4 + ?E*12 + ?F*2 + ?G*3 + ?H*2 + ?I*9 + ?J +
+      ?K + ?L*4 + ?M*2 + ?N*6 + ?O*8 + ?P*2 + ?Q*1 + ?R*6 + ?S*4 + ?T*6 + ?U*4 +
+      ?V*2 + ?W*2 + ?X*1 + ?Y*2 + ?Z*1).split("").shuffle
```

Comment:
As discussed in class this is probably fine for the Scrabble project and in future projects we'll have our static (and dynamic) data defined elsewhere outside of our code files.
However, in general it's a good (taste/style) rule to avoid using neat or quirky parts of a language just because they're available. The more time that another reader of your code must take to follow it, perhaps because you're using a language feature they've not encountered before, the more difficult it will be for them to understand your intention.

Also as a very, very minor side note: Due to the way that Ruby used to characters in 1.8 and prior versions, this code would not operate as intended on those versions. We will in the future see ways that you can build your projects to indicate a specific preferred Ruby version.